packages/libs/react-components/src/components/IconButton/IconButton.tsx

```diff
@@ -29,13 +32,19 @@ const Button = styled('button', {
 export const IconButton: FC<IIConButtonProps> = ({
   icon,
   onClick,
+  color = 'transparent',
```

Comment:
I believe this does not work with the type since "transparent" is not a key of colors. As we discussed earlier, we will need to come to a group consensus on how strict we want our components, but for the sake of consistency I would suggest changing the type to `PropertyValue` like we currently have in GridItem 
```export interface IGridItemProps {
  children?: ReactNode;
  colStart?: ColSpanType;
  colEnd?: ColSpanType;
  rowStart?: ColSpanType;
  rowEnd?: ColSpanType;
  area?: CSSProperties['gridArea'];
  bg?: PropertyValue<'backgroundColor', typeof config>;
}

Reply:
yeah you are right...sorry. consistency is key here

Reply:
will change it with the icons now as well then, because I copied it from there :)

Reply:
even better.. the `color` is already an attribute in `React.HTMLAttributes<HTMLButtonElement>`, which we extend. problem solved

Reply:
Yes, but I think it inherits the type without our stitches tokens. If that's the case, I think defining it is necessary to override that type with a type that includes the tokens